contracts/hardhat.config.ts

```diff
@@ -4,20 +4,54 @@ import "@nomiclabs/hardhat-waffle";
  * @type import('hardhat/config').HardhatUserConfig
  */
 export default {
-  solidity: "0.8.4",
   paths: {
     artifacts: "./artifacts",
     cache: "./cache",
     sources: "./src",
     tests: "./test",
   },
-  overrides: {
-    "src/v0.8/*": {
-      version: "0.8.4",
-      optimizer: {
-        enabled: true,
-        runs: 200,
-      },
+  networks: {
+    hardhat: {
+      allowUnlimitedContractSize: true,
```

Comment:
Any reason not to turn this off? Seems like we want to run into this at compile time rather than wait for it to fail in tests.

Reply:
Unfortunately, `FluxAggregator` is too large to deploy during tests. I'm not sure how we get around this since we aren't changing that code anymore.

Reply:
It didn't used to be too large. What changed?

Reply:
Maybe the hardhat tests don't run the optimizer before testing the contracts? I'll look into it.

Reply:
Fixed! It turns out the optimizer settings weren't being applied properly due to a typo in this config file 🤦